### DISCUSSION TITLE
ABB Multimove WaitSyncTask

### DISCUSSION THREAD

**Discussion by gegardne** [visose/Robots#153] [Q&A] [answered]
Hi, I am trying to understand how the plugin works with ABB multimove. I've put together a multimove program for 2 IRB 2600s. I've modified one of the example xml files and am able to simulate the 2 robots and generate programs for T_ROB 1 and 2.  I see that the program requires both input toolpaths to have the same number of targets and creates  SyncMoveOn and SyncMoveOff commands at the beginning and end of the proc. I was wondering if there is a way to use a WaitSyncTask command and simulate it in grasshopper? I am able to insert the command using the custom command, but it doesn't affect the simulation.

**Comment by visose** [comment, marked as answer]
`WaitSyncTask`  is not supported directly by the plugin, but a workaround you can use is to just repeat the same target on the static robot for every target that the other robot is moving. 

So you end up with two lists of targets of the same length, only that some targets will be duplicates to fill the blanks when the robot is not moving.
